### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by samuko-things**
**This pull request makes the following changes:**
* Updated pull request template for easy pr creation

**Checklist:**
- [x] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me
- [x] I certify that I ran my checklist.

Ping @code-collabo/node-mongo-cli

### CODE REVIEW

**Comment by samuko-things** [comment]
@Ifycode, i just updated the pr template, please help review it

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,14 +1,9 @@
 **This pull request makes the following changes:**
-* Fixes issue code-collabo/node-mongo-cli#issueNumberHere 
+* Fixes issue #issueNumberHere
```
Comment (Ifycode) [inline .github/pull_request_template.md:2]: Same thing here too. Change this line to:
* Fixes issue code-collabo/repo-name-here#issueNumberHere

**Review by Ifycode — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (samuko-things) [reply .github/pull_request_template.md]: I have made the change

**Comment by samuko-things** [comment]
I have made the change

**Review by Ifycode — APPROVED** [review]

**Pull request merged**